### PR TITLE
Add Home View & Skeleton Views

### PR DESCRIPTION
**Pull request by Nitish145**
### This PR contains the following changes:

- Add assets for home view.
- Add home view and related components.
- Update folder structure to follows subdirectories.

### Screenshots / Video Recording
ezgif com-video-to-gif

### CODE REVIEW

**Comment by Nitish145** [comment]
@tachyons please check !!

**Comment by tachyons** [comment]
Can you add screen shot to PR ?

**Comment by Nitish145** [comment]
> Can you add screen shot to PR ?

Yes Sure!

**Comment by Nitish145** [comment]
> > Can you add screen shot to PR ?
> 
> Yes Sure!

**Comment by Nitish145** [comment]
@tachyons are any more changes required here ?

File: android/app/build.gradle [hunk 1]
```diff
@@ -37,8 +37,7 @@ android {
     }
 
     defaultConfig {
-        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.mobile_app"
+        applicationId "com.circuitverse.mobile_app"
```
Comment (tachyons) [inline android/app/build.gradle:40]:
```suggestion
        applicationId "org.circuitverse.mobile_app"
```

**Comment by tachyons** [comment]
@nitin10s Can you please review ux aspects

File: lib/ui/views/home/home_view.dart [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          padding: const EdgeInsets.all(16),
+          child: Column(
+            children: <Widget>[
+              _buildHeader(),
+              _buildHomePageSketch(),
+              _buildTeachersAndContributorButtons(),
+              CVSubheader(
+                title: "Features",
+                subtitle:
+                    "Design circuits quickly and easily with a modern and intuitive user interface with drag-and-drop, copy/paste, zoom and more.",
```
Comment (tachyons) [inline lib/ui/views/home/home_view.dart:73]: Can we make this text translatable using i18n ?

Comment (Nitish145) [reply lib/ui/views/home/home_view.dart:73]: Yes, so we need to setup internationalisation based on user's device preferences or we must provide them with the options to change language too? @tachyons

Comment (tachyons) [reply lib/ui/views/home/home_view.dart:73]: We don't user preference for the locale yet, so use system locale

Comment (Nitish145) [reply lib/ui/views/home/home_view.dart:73]: Sure!

**Comment by nitin10s** [comment]
Yes, I was planning to, there needs a lot of refining to be done. I think we should set up a call for it.

**Comment by Nitish145** [comment]
@tachyons please review this..

**Pull request merged**